src/index.css

```diff
@@ -43,6 +43,21 @@ body {
   display: none;
 }
 
+/*
+ Screen reader only label, hidden for normal users
+*/
+
+.sr-only-label {
```

Comment:
Try to implement this style in `tailwind css`

Reply:
Done!